plymouth-0.9.4-smooth-transistion-support.patch

```diff
@@ -0,0 +1,67 @@
+diff -Naur plymouth-0.9.4-20200314/configure.ac plymouth-0.9.4-20200314.tpg/configure.ac
+--- plymouth-0.9.4-20200314/configure.ac	2020-03-11 19:23:44.000000000 +0000
++++ plymouth-0.9.4-20200314.tpg/configure.ac	2020-03-14 21:15:18.788364963 +0000
+@@ -336,6 +336,7 @@
+ 	   scripts/plymouth-populate-initrd
+ 	   scripts/plymouth-set-default-theme
+            scripts/Makefile
++           systemd-units/plymouth-deactivate.service
+            systemd-units/plymouth-halt.service
+            systemd-units/plymouth-kexec.service
+            systemd-units/plymouth-poweroff.service
+diff -Naur plymouth-0.9.4-20200314/systemd-units/Makefile.am plymouth-0.9.4-20200314.tpg/systemd-units/Makefile.am
+--- plymouth-0.9.4-20200314/systemd-units/Makefile.am	2020-03-11 19:23:44.000000000 +0000
++++ plymouth-0.9.4-20200314.tpg/systemd-units/Makefile.am	2020-03-14 21:20:07.749825763 +0000
+@@ -1,4 +1,5 @@
+ systemd_unit_templates =                                                     \
++        plymouth-deactivate.service.in                                       \
+         plymouth-switch-root.service.in                                      \
+         plymouth-start.service.in                                            \
+         plymouth-read-write.service.in                                       \
+@@ -55,7 +56,8 @@
+ 	(cd $(DESTDIR)$(SYSTEMD_UNIT_DIR)/sysinit.target.wants &&            \
+ 		rm -f plymouth-start.service plymouth-read-write.service)
+ 	(cd $(DESTDIR)$(SYSTEMD_UNIT_DIR)/multi-user.target.wants &&         \
+-		rm -f plymouth-quit.service plymouth-quit-wait.service)
++		rm -f plymouth-quit.service plymouth-quit-wait.service &&    \
++		plymouth-deactivate.service)
```

Comment:
This looks wrong, it's trying to execute plymouth-deactivate.service, which doesn't look like an executable... ("rm -f ... && plymouth-deactivate.service")

Doesn't look like something that'll cause us any harm given it seems to be in a "make uninstall" target, but nevertheless...

Reply:
Thanks fixed